### PR TITLE
Implementation of hourly forecasts

### PR DESCRIPTION
**Pull request by Mnoble-19**
**_WHAT DOES THIS PR DO?_**
* updates the training and prediction job for the hourly forecasts model
* Fixes issue of missing forecasts for some sites
* Code fixes for daily forecast job as well
* Add saving data to BigQuery
* Remove redudant code

**_HOW DO I TEST OUT THIS PR?_**
hourly & daily forecast endpoints

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: src/predict/jobs/forecast/Dockerfile [hunk 1]
```diff
@@ -14,13 +14,16 @@ RUN pip install -r requirements.txt
 FROM base as production
 ENV ENV=production
 CMD ["python", "daily_forecasts.py"]
+CMD ["python", "hourly_forecasts.py"]
 
 # Staging
 FROM base as staging
 ENV ENV=staging
 CMD ["python", "daily_forecasts.py"]
+CMD ["python", "hourly_forecasts.py"]
```
Comment (Mnoble-19) [inline src/predict/jobs/forecast/Dockerfile:23]: Hi @NoahNsimbe and @123MwanjeMike. I need your help here. These 2 scripts are separate jobs and ought to run at different times. Is this okay? How does one discern between them which one to run at a given period

Comment (NoahNsimbe) [reply src/predict/jobs/forecast/Dockerfile:30]: @123MwanjeMike

Comment (123MwanjeMike) [reply src/predict/jobs/forecast/Dockerfile:30]: Hi @Mnoble-19 
Why are you having two different entry points(`CMD`) in the same stage dockerfile.
Can we use different stages for this.
https://docs.docker.com/build/building/multi-stage/

**Review by 123MwanjeMike — CHANGES_REQUESTED** [review]

File: src/predict/jobs/forecast/Dockerfile [hunk 2]
```diff
@@ -11,16 +11,29 @@ RUN pip install --upgrade pip
 RUN pip install -r requirements.txt
 
 # Production
```
Comment (123MwanjeMike) [inline src/predict/jobs/forecast/Dockerfile:13]: Thanks @Mnoble-19 
Could we also have these comments updated to something like “production daily” and “production hourly”

**Review by 123MwanjeMike — APPROVED** [review]

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**